doc/source/whatsnew/v1.2.1.rst

```diff
@@ -20,6 +20,7 @@ Fixed regressions
 - Bug in repr of float-like strings of an ``object`` dtype having trailing 0's truncated after the decimal (:issue:`38708`)
 - Fixed regression in :meth:`DataFrame.groupby()` with :class:`Categorical` grouping column not showing unused categories for ``grouped.indices`` (:issue:`38642`)
 - Fixed regression in :meth:`DataFrame.any` and :meth:`DataFrame.all` not returning a result for tz-aware ``datetime64`` columns (:issue:`38723`)
+- :func:`read_excel` does not work for non-rawbyte file handles (issue:`38788`)
```

Comment:
I expect maybe a copy and paste error, should have been `:issue:`

Reply:
fixed in #38850

Reply:
thank you!